### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by yuxiang-zhang**
### Description

This PR is related to https://github.com/weaveworks/eksctl-private/issues/833. 

- Adding instructions for installs on Windows platforms

For security purposes:
- Updating installation instructions to include checksum verification
- Guidelines for third-party installers

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by A-Hilaly — APPROVED** [review]
Looks good for me👍 
One tiny nit below

File: README.md [hunk 1]
```diff
@@ -12,38 +12,62 @@ _Need help? Join [Weave Community Slack][slackjoin]._
...
+### For Unix
 To download the latest release, run:
 
-```
-curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
-sudo mv /tmp/eksctl /usr/local/bin
-```
+```sh
+# for ARM systems, set ARCH to: armv6, armv7 or arm64
+ARCH=amd64
```
Comment (A-Hilaly) [inline README.md:22]: looks like Github didn't send my comment or i missed something but here is a little suggestion
```suggestion
ARCH=$(uname -m) # or $(go env GOARCH)
```

Comment (yuxiang-zhang) [reply README.md:22]: Nice one! The `$(uname -m)` gets expanded to `x86_64` on my `amd64` machines. I think we could go with the latter.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # `eksctl` - The official CLI for Amazon EKS
 
-[![Coverage Status](https://coveralls.io/repos/github/weaveworks/eksctl/badge.svg?branch=main)](https://coveralls.io/github/weaveworks/eksctl?branch=main) [![Go Report Card](https://goreportcard.com/badge/github.com/weaveworks/eksctl)](https://goreportcard.com/report/github.com/weaveworks/eksctl)
+[![Go Report Card](https://goreportcard.com/badge/github.com/weaveworks/eksctl)](https://goreportcard.com/report/github.com/weaveworks/eksctl)
```
Comment (Himangini) [inline README.md:3]: ⭐ ty for removing this.

**Review by Himangini — APPROVED** [review]
✨ nicely done 👍🏻

**Review by azpaulp — APPROVED** [review]

**Pull request merged**